Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -460,57 +460,58 @@ If the set point is changed during an autotuning process, a warning will be
 generated. The ongoing tuning process will be halted, and no adjustments will be made
 to the PID parameters.
 </li>
+<li>
+The autotuning must be conducted after the initialization is completed.
```

Comment:
What is the "initialization"? How can the user, or the controller, avoid the autotuning being conducted during the initialization?